js/src/tooltip.js

```diff
@@ -468,12 +483,11 @@ const Tooltip = (() => {
 
       triggers.forEach((trigger) => {
         if (trigger === 'click') {
-          $(this.element).on(
+          EventHandler.on(this.element,
             this.constructor.Event.CLICK,
-            this.config.selector,
+            this.config.selector || '',
```

Comment:
why ?

Reply:
`.on` method explicitly checks for empty string, but selector can be `undefined` or `null`
Probably the check event handler should be modified